### ISSUE TITLE
How to run mix on docker container

### ISSUE THREAD

**Issue by johncadengo** [papercups-io/papercups#673]
I'm using the docker container and I'm trying to run this command:
```
    # NB: if you're self-hosting, you can run the following to upgrade your account:
     # ```
     # $ mix set_subscription_plan [YOUR_ACCOUNT_TOKEN] team
     # ```
```

This comes from [accounts.ex](https://github.com/papercups-io/papercups/pull/447/files#diff-ef913e27ecc7b178263bc558980e8e994fc57f1504b0598fa2f887bc563fac05R145). How do I run this command from within the docker container?

**Comment by cheeseblubber** [comment]
@johncadengo hey John this is something I need to fix in the docker file. We're looking to get this fix this week.

**Comment by johncadengo** [comment]
@cheeseblubber Thanks for the update. Just checking in on this. Are you all still supporting docker? (You mentioned moving away from Docker in the other issue: https://github.com/papercups-io/papercups/issues/672) It would be great if I could run this command in my docker container, so I'm looking forward to the update from you.

**Comment by cheeseblubber** [comment]
We're going to enable it by default https://github.com/papercups-io/papercups/issues/692. This would fix your issue right? @johncadengo

**Comment by johncadengo** [comment]
@cheeseblubber yes https://github.com/papercups-io/papercups/issues/692 would fix my issue. I don't see a need to run mix within a docker container beyond this command. Thanks.
